projects/storefrontlib/cms-components/content/tab-paragraph-container/tab-paragraph-container.component.ts

```diff
@@ -144,7 +150,10 @@ export class TabParagraphContainerComponent implements AfterViewInit, OnInit {
     }
 
     // Render the tabs after the templates have completed loading in the view.
-    this.tabs$ = combineLatest([this.components$, this.tabRefs.changes]).pipe(
+    this.tabs$ = combineLatest([
+      this.components$,
+      this.tabRefs.changes.pipe(startWith(this.tabRefs)),
```

Comment:
After navigating to different route and back the `tabRefs` are already initialised with values and `this.tabRefs.changes` does not emit any value. This change ensures that at least the initial value will be taken into account. This solves the problem

Reply:
Nice find! Are we able to add an e2e test for the empty order history case just to make sure we have coverage?